lib/reporting/drop_off_report.rb

```diff
@@ -223,15 +223,15 @@ def dropoff_metrics_table
         ],
         [
           'Verified (event)',
-          idv_personal_key_submitted,
+          idv_final_resolution_verified,
```

Comment:
This event will only show you users who verify with a phone number and document capture and are not marked for fraud review. People who go into those out-of-band flows (verify-by-mail, in-person-proofing, and fraud review) will not be counted by this even if they are eventually verified.

The current event suffers from a similar drawback: It does not count users who do verify-by-mail and eventually proof because they do not see the personal key screen until after they finish; that could be a long ways out. That does make it an improvement and I think this change is good as long as you understand the caveats.

Reply:
Then, does `ipp_verification_total` not do what the name says? It looks like we can get `data[Events::IPP_ENROLLMENT_UPDATE]` where `passed: false`, so simply counting those events doesn't provide an accurate verification total.
Unless there's data cleaning on the CW side?